internal/pkg/v2/utils/address.go

```diff
@@ -0,0 +1,121 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/models"
+)
+
+const (
+	ContentTypeKey       = "Content-Type"
+	ContentTypeJsonValue = "application/json; charset=utf-8"
```

Comment:
is the charset necessary?
please use the one in go-mod-core-contracts
https://github.com/edgexfoundry/go-mod-core-contracts/blob/205e078090c85c2987ffa64a69d75c76dffd8e81/clients/constants.go#L79

Reply:
Modified. The charset is not necessary.